### ISSUE TITLE
Wrong percentage of the task progress

### ISSUE THREAD

**Issue by ostasevych** [nextcloud/tasks#2316]
Please, make the field showing the progress of a task execution wider. Now it can contain only 1 digit.

So, when editing I see only 1-digit figure:
![image](https://github.com/nextcloud/tasks/assets/5199030/e8b86320-5cb0-47e5-81b3-4199df8ed61c)

And when have edited I can see the 2-digit figure:
![image](https://github.com/nextcloud/tasks/assets/5199030/36d11807-b6e3-4e65-95c1-e6465e0777a8)

**Comment by raimund-schluessler** [comment]
Duplicate of #2304.

**Comment by ostasevych** [comment]
Both the data in task priority and percentage of performing windows are cut and only the first digit is shown, what brings inconvenience when defining the level.

**Comment by oleua** [comment]
The issue is still not solved

![Image](https://github.com/user-attachments/assets/32d6b3bc-65c9-420b-a5a4-a3237600292c)

The solution - replace in `tasks/css/main-C80Xy9ML.chunk.css`:
```
input[type=number][data-v-6f9b610d]{width:50px; flex-basis;50px;
```
with
```
input[type=number][data-v-6f9b610d]{width:60px; flex-basis;60px;
```
So, it will look:

![Image](https://github.com/user-attachments/assets/765fd6d1-4464-4187-9dc2-b6daf814196c)

**Comment by raimund-schluessler** [comment]
It is solved in `main` but not released yet. Will be released with 0.17.0.
